### PR TITLE
Add deprecation annotations to ported utils

### PR DESCRIPTION
**Pull request by brandenrodgers**
## Description and Context

Adding deprecation annotations to exported utils that the HubSpot CLI is now importing from the new local-dev-lib library. This should help to prevent new changes from being made in this library for exports that aren't being used anymore.

### CODE REVIEW

File: lib/walk.js [hunk 1]
```diff
@@ -28,6 +28,11 @@ const generateRecursiveFilePromise = async (dir, file) => {
   });
 };
 
+/**
+ * @deprecated
+ * Use the corresponding export from local-dev-lib
+ * https://github.com/HubSpot/hubspot-local-dev-lib
+ */
```
Comment (camden11) [inline lib/walk.js:35]: Might be helpful to add the new import paths for those that have changed. In this case its `@hubspot/local-dev-lib/fs`

File: lib/files.js [hunk 2]
```diff
@@ -4,6 +4,11 @@ const { getEnv } = require('@hubspot/local-dev-lib/config');
 const { getHubSpotWebsiteOrigin } = require('./urls');
 const { ENVIRONMENTS } = require('./constants');
 
+/**
+ * @deprecated
+ * Use the corresponding export from local-dev-lib
+ * https://github.com/HubSpot/hubspot-local-dev-lib
+ */
```
Comment (camden11) [inline lib/files.js:11]: renamed to `@hubspot/local-dev-lib/cms/themes`

**Review by camden11 — COMMENTED** [review]
There's a few places where it might be helpful to indicate the new import path. I also noticed there's a few more places where we'll need to add the annotation:
* `git.js`
* `sandboxes.js`
* `trackUsage` within `filemapper.js` (new import path: `@hubspot/local-dev-lib/trackUsage`)

File: lib/files.js (hunk 2 above)
Comment (brandenrodgers) [reply lib/files.js:11]: added in [a821417](https://github.com/HubSpot/cli-lib/pull/39/commits/a821417f90003cb28b571909a63f011d96365049)

File: lib/walk.js (hunk 1 above)
Comment (brandenrodgers) [reply lib/walk.js:35]: Good idea. Added in [a821417](https://github.com/HubSpot/cli-lib/pull/39/commits/a821417f90003cb28b571909a63f011d96365049)

**Comment by brandenrodgers** [comment]
@camden11 I added deprecation annotations to the sandboxes and trackUsage exports, but it looks like the CLI is still using the git utils exported from cli-lib.

**Review by camden11 — APPROVED** [review]
@brandenrodgers it's not using the github utils, but confusingly there's a separate file called `git.js` that it is using. (this has been renamed to `gitignore.ts` in local dev lib) Approving but just make sure to add that final annotaiton before you merge!

**Pull request merged**